packages/@aws-cdk/aws-kinesisanalytics-flink-alpha/test/application.test.ts

```diff
@@ -21,21 +21,21 @@ describe('Application', () => {
     stack = new core.Stack();
     bucket = new s3.Bucket(stack, 'CodeBucket');
     requiredProps = {
-      runtime: flink.Runtime.FLINK_1_11,
+      runtime: flink.Runtime.FLINK_1_19,
```

Comment:
nit: For future references, I would prefer to have this as parameterized tests to cover both existing and new runtime.